Cargo.toml

```diff
@@ -2,10 +2,10 @@
 
 members = [
 	"dataplane",
+	"errno",
```

Comment:
Why did the location of this move?

Reply:
the linter put them in alphabetical order :woman_shrugging:

Reply:
I'm happy to take that out of the pr, it just seems harmless

Reply:
It is specifically not alphabetical, which is why I asked, I don't care either way, was just curious.

Reply:
hmmm, you are right.  I always just let the linter do its thing but it looks like it went off the rails here.  Wil fix